### PR TITLE
Add arguments to lockedgen for using it in other places

### PR DESCRIPTION
**Pull request by egonelbre**

### CODE REVIEW

File: scripts/lockedgen/main.go [hunk 1]
```diff
@@ -20,20 +23,35 @@ import (
 
 func main() {
```
Comment (mniewrzal) [inline scripts/lockedgen/main.go]: Can you add `// +build ignore` for this script?

Comment (egonelbre) [reply scripts/lockedgen/main.go]: Not sure why?

Comment (mniewrzal) [reply scripts/lockedgen/main.go]: just to avoid having one more executable `lockedgen` on path. Like other scripts in this folder.

Comment (egonelbre) [reply scripts/lockedgen/main.go]: anyways did the move and added tag

**Pull request merged**